interface/python_binding/pyktx/vk_format.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2023, Shukant Pal and Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from enum import IntEnum
+
+
+class VkFormat(IntEnum):
```

Comment:
Must have been painful creating this. It should probably be generated from the same mkvkformatfiles awk script that generates `lib/vkformat_enum.h`.

Reply:
I copied the Java VkFormats file and did some find/replace. Is it necessary to use awk, since these are static files?